Fix:

error: incompatible function pointer types initializing 'const void (*)(uint16_t, const char *)' (aka 'const void (*)(unsigned short, const char *)') with an expression of type 'void (uint16_t, const char *)' (aka 'void (unsigned short, const char *)') [-Wincompatible-function-pointer-types]
error: incompatible function pointer types initializing 'const void (*)(void)' with an expression of type 'void (void)' [-Wincompatible-function-pointer-types]
error: incompatible function pointer types initializing 'const int (*)(void)' with an expression of type 'int (void)' [-Wincompatible-function-pointer-types]
error: incompatible function pointer types initializing 'const void (*)(void)' with an expression of type 'void (void)' [-Wincompatible-function-pointer-types]

https://github.com/mywave82/opencubicplayer/issues/105
https://github.com/mywave82/opencubicplayer/commit/638a348aae972471dc5c5f12897b2cf37d637951
--- filesel/filesystem-file-dev.h
+++ filesel/filesystem-file-dev.h
@@ -27,11 +27,11 @@ struct DevInterfaceAPI_t
 	const struct dmDrive         *dmFile;
 #endif
 
-	const void (*KeyHelp) (uint16_t key, const char *shorthelp); /* Called on ALT-K to issue help about each keyboard shortcut */
-	const void (*KeyHelpClear) (void); /* Clears the current keyboard shortcut list, only used by keyboard/display loops */
-	const int  (*KeyHelpDisplay) (void); /* Draws the keyboard shortcut list and polls keyboard. Call for each draw-iteration until it returns zero */
+	void (*KeyHelp) (uint16_t key, const char *shorthelp); /* Called on ALT-K to issue help about each keyboard shortcut */
+	void (*KeyHelpClear) (void); /* Clears the current keyboard shortcut list, only used by keyboard/display loops */
+	int  (*KeyHelpDisplay) (void); /* Draws the keyboard shortcut list and polls keyboard. Call for each draw-iteration until it returns zero */
 
-	const void (*fsDraw) (void); /* Draws the filesystem browser, great for virtual devices that has a dialog */
+	void (*fsDraw) (void); /* Draws the filesystem browser, great for virtual devices that has a dialog */
 };
 
 struct IOCTL_DevInterface
